content/en/developers/community/libraries.md

```diff
@@ -198,7 +198,7 @@ If you've written a Datadog library and would like to add it to this page, send
 [27]: https://github.com/keirans/datadog-management
 [28]: https://github.com/zendesk/consul2dogstats
 [29]: https://github.com/cvent/dogscaler
-[30]: https://github.com/Dynatrace/Dynatrace-AppMon-Datadog-Plugin
+[30]: https://github.com/sworisbreathing/Dynatrace-Datadog-Plugin
```

Comment:
looks like this repository has been archived and may be deprecated, are we comfortable with that?

Reply:
also the link to the repo website https://community.dynatrace.com/community/display/DL/Datadog+Plugin looks down (it redirects to confluence.dynatrace.com which resolves to a private IP)

Reply:
let's remove this plugin from the list. If someone confirms another Dynatrace library, we can add it back
```suggestion
```